### PR TITLE
Metadata review - WFP-PARADK updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updates to WFP PARADK cal csvs per metadata review project.

### CODE REVIEW

File: calibration/PARADK/CGINS-PARADK-20423__20140829.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
 20423,CC_dark_offset,1.2,
-20423,CC_scale_wet,9.96e-18,
+20423,CC_scale_wet,9.85e-18,
```
Comment (akjensen7) [inline calibration/PARADK/CGINS-PARADK-20423__20140829.csv]: Looks good.

Comment (reedan88) [reply calibration/PARADK/CGINS-PARADK-20423__20140829.csv]: Looks good

File: calibration/PARADK/CGINS-PARADK-20436__20150812.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20436,CC_dark_offset,1.5,
+20436,CC_scale_wet,9.92e-18,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20436__20150812.csv]: CC_scale_wet should be 1.03e-17 and CC_dark_offset 1.2

File: calibration/PARADK/CGINS-PARADK-20437__20140829.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20437,CC_dark_offset,1.1,
+20437,CC_scale_wet,9.90e-18,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20437__20140829.csv]: Looks good

File: calibration/PARADK/CGINS-PARADK-20447__20121217.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20447,CC_dark_offset,1.2,
+20447,CC_scale_wet,1.00E-17,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20447__20121217.csv]: Looks good

File: calibration/PARADK/CGINS-PARADK-20447__20140829.csv [hunk 5]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20447,CC_dark_offset,1.1,
+20447,CC_scale_wet,9.96E-18,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20447__20140829.csv]: Looks good

File: calibration/PARADK/CGINS-PARADK-20447__20140923.csv [hunk 6]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20447,CC_dark_offset,1.2,
+20447,CC_scale_wet,1.01E-17,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20447__20140923.csv]: Looks good

File: calibration/PARADK/CGINS-PARADK-20447__20150311.csv [hunk 7]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20447,CC_dark_offset,1.1,
+20447,CC_scale_wet,9.84E-18,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20447__20150311.csv]: Looks good

File: calibration/PARADK/CGINS-PARADK-20447__20160128.csv [hunk 8]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20447,CC_dark_offset,0.9,
+20447,CC_scale_wet,1.13E-17,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20447__20160128.csv]: Looks good

File: calibration/PARADK/CGINS-PARADK-20449__20150812.csv [hunk 9]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20449,CC_dark_offset,1.4,
+20449,CC_scale_wet,1.00e-17,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20449__20150812.csv]: Looks good

File: calibration/PARADK/CGINS-PARADK-20464__20130924.csv [hunk 10]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
 20464,CC_dark_offset,1.1,
-20464,CC_scale_wet,9.96e-18,
+20464,CC_scale_wet,1.01e-17,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20464__20130924.csv]: Looks good.

File: calibration/PARADK/CGINS-PARADK-20466__20130924.csv [hunk 11]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20466,CC_dark_offset,1.0,
+20466,CC_scale_wet,1.00e-17,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20466__20130924.csv]: Looks good

File: calibration/PARADK/CGINS-PARADK-20467__20130924.csv [hunk 12]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20467,CC_dark_offset,1.0,
+20467,CC_scale_wet,1.01e-17,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20467__20130924.csv]: Looks good.

File: calibration/PARADK/CGINS-PARADK-20468__20130924.csv [hunk 13]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
 20468,CC_dark_offset,1.0,
-20468,CC_scale_wet,9.81e-18,
+20468,CC_scale_wet,1.02e-17,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20468__20130924.csv]: Looks good.

File: calibration/PARADK/CGINS-PARADK-20436__20150812.csv (hunk 2 above)
Comment (rtravis-whoi) [reply calibration/PARADK/CGINS-PARADK-20436__20150812.csv]: fixed

**Pull request merged**